<launch>
    <arg name="config" default="demo"/>
    <rosparam command="load" file="$(find lasr_skills)/config/$(arg config).yaml" />
    <node pkg="lasr_skills" type="go_to_location.py" name="go_to_location" output="screen" />
<!--    <node pkg="lasr_skills" type="go_to_semantic_location.py" name="go_to_semantic_location" output="screen" />-->
</launch>
